src/Umbraco.Web.UI.Client/src/views/common/overlays/user/user.html

```diff
@@ -78,8 +77,8 @@ <h5>
     </div>
 
 
-    <div class="umb-control-group" ng-if="!showPasswordFields">
-        <h5><localize key="user_yourHistory" /></h5>
+    <div class="umb-control-group" ng-if="!showPasswordFields && history.length">
+        <h5><localize key="user_yourHistory" /></h5
```

Comment:
Hi @BatJan ,

The h5 is missing the closing tag. Can you fix? Apart from that the PR works as expected :-)

Poornima

Reply:
Ouch! Good catch - It's been fixed now 😃